recipes/pandas-gbq/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: pandas-gbq
+  version: {{ version }}
+
+source:
+  fn: pandas-gbq-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/source/p/pandas-gbq/pandas-gbq-{{ version }}.tar.gz
```

Comment:
We've normally been going to `pypi.org` (Warehouse) to download packages as other methods of acquiring the source from PyPI have gotten messed up. Please see the [example]( https://github.com/conda-forge/staged-recipes/blob/6b75c52ad1dfb1d8144247f52fe042d98a1d8336/recipes/example/meta.yaml ) for details.